io.github.yuxshao.ptcollab.yml

```diff
@@ -0,0 +1,31 @@
+id: io.github.yuxshao.ptcollab
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ptcollab
+modules:
+  - name: ptcollab
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://github.com/yuxshao/ptcollab.git
+        tag: "v0.6.4.8"
+        commit: "5feb27decb9e0600d1defdaf9dc1424c6fdcb7ad"
+  - name: share
+    buildsystem: simple
+    sources:
+      - type: dir
+        path: share
+    build-commands:
+      - rm /app/share/applications -rf
+      - rm /app/share/icons/hicolor/*/*/*
+      - cp -rf ../share /app/
+finish-args:
```

Comment:
this section should be found above `modules`.

Reply:
Ok, done